modules/ldap-syncing-spec.adoc

```diff
@@ -0,0 +1,250 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-syncing-spec_{context}"]
+= LDAP sync configuration specification
+
+The object specification for the configuration file is below.  Note that the different schema
+objects have different fields.  For example, v1.ActiveDirectoryConfig has no `groupsQuery`
+field whereas v1.RFC2307Config and v1.AugmentedActiveDirectoryConfig both do.
+
+[IMPORTANT]
+====
+There is no support for binary attributes. All attribute data coming from the
+LDAP server must be in the format of a UTF-8 encoded string. For example, never
+use a binary attribute, such as `objectGUID`, as an ID attribute. You must use
+string attributes, such as `sAMAccountName` or `userPrincipalName`, instead.
+====
+
+[[sync-ldap-v1-ldapsyncconfig]]
+== v1.LDAPSyncConfig
+
+`LDAPSyncConfig` holds the necessary configuration options to define an LDAP
+group sync.
+
+[options="header"]
+|===
+|Name |Description |Schema
+
+|`kind`
+|String value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: link:https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#types-kinds[]
```

Comment:
I know this was already like this in 3.11, but I don't like the "More info:" thing. I would make it into a sentence and not show the whole URL here. (applicable in a bunch of places in these tables).